services/provider/server/server.go

```diff
@@ -355,7 +357,7 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 	for _, cephRes := range consumerResource.Status.CephResources {
```

Comment:
As all of the storage consumer direct resources are known ahead of time and the list is static, why do we need to go over the range if we already know need to be created?

My thinking is to remove the whole section and just add the two remaining resources, the health checker client secret and the mon-secret

Reply:
mon secret is already fetched above i think you meant only health check client.